main.js

```diff
@@ -14,6 +14,8 @@ const configs = {
     tweetsPerPage: 10,
     timeoutBetweenTweetsScraped: 5000,
     puppeteerLaunchOptions: { headless: false },
+    nextLabel: 'Siguiente',
+    loginLabel: 'Iniciar sesión'
```

Comment:
Can you leave the English labels for now? On a follow up we can have a default config file on version control (with English), and then a gitgnored config file can override it with different labels. Can also add a comment in the readme about spanish support.